website/content/docs/concepts/password-policies.mdx

```diff
@@ -360,6 +360,39 @@ This policy generates 8 character passwords from the charset `abcde01234` and re
 character from `01234` to be in it, but does not require any characters from `abcde`. The password
 `04031945` may result from this policy, even though no alphabetical characters are in it.
 
+## Default Password Policy
+
+Vault ships with a Default Password Policy and this gets applied to any password that Vault 
+generates where a custom password policy is not explicitly set. The default password policy is 
+defined below.
+
+<details>
+<summary>20 characters with at least 1 uppercase, 1 lowercase, 1 number, and 1 dash character.</summary>
```

Comment:
```suggestion
```
Folded into the previous section. Also, I'm not sure what the `details` and `summary` tags are for? I don't think the markdown parser recognizes those tags as valid formatting